deposit-transactions.md

```diff
@@ -0,0 +1,165 @@
+# Deposit Transactions
+
+Although the L2 execution engine is nearly identical to that of L1, an important difference is the
+introduction of a new 'Deposit' Transaction Type. Deposit transactions have the following notable
+distinctions from existing transaction types:
+
+1. They are initiated by the System as part of the protocol.
+2. Do not require signature validation.
+
+## Deposit Transaction Type
+
+We define a new [EIP-2718] compatible transaction type with the prefix `0x7E`, and the following
+fields:
+
+[EIP-2718]: <https://eips.ethereum.org/EIPS/eip-2718>
+
+<!-- ToDo: set to more GoLang like type defs? -->
+- `address to`
+- `address from`
+- `uint256 value`
+- `bytes data`
+
+This transaction type contains a subset of the fields used in [EIP-155], but does not include
+signature information.
+
+[EIP-155]: https://eips.ethereum.org/EIPS/eip-155
+
+Although in practice we define only one new Transaction Type we can distinguish between two distinct
+transactions which occur in the deposit block, based on their positioning. The first transaction
+MUST be the [L1 Attributes Deposit Transaction][l1-attributes-deposit-transaction], followed by a
+dynamic array of [Deposited Transactions][deposited-transactions] submitted to the Deposit Feed
+contract by accounts on L1.
+
+## L1 Attributes Deposit Transaction
+
+[l1-attributes-deposit-transaction]: #l1-attributes-deposit-transaction
+
+This transaction is a call to the [Layer 1 Attributes Predeploy][l1-attributes-predeploy] contract.
+
+This transaction MUST have the following values:
+
+1. `from` is the Depositor Account `0xdeaddeaddeaddeaddeaddeaddeaddeaddead0001`.
+1. `to` is `0x4200000000000000000000000000000000000014` (the address of the L1 attributes predeploy
+   contract).
+1. `value` is `0`
```

Comment:
Does this imply that moving ETH in and out of the rollup is out-of-scope for deposit-only rollup?

Reply:
This only applies to the first deposit block transaction which submits the L1 attributes.

HOWEVER, it is true that we have not yet specified a method for withdrawing ETH. 
That would look a bit weird, because it would require submitting a transaction to the Deposit Feed contract on L1, in order to burn ETH on L2, and then the Feed contract would transfer the value.
But it's also odd to have ETH permanently locked in the Feed. 🤔

Reply:
I'm going to add a TODO for now.